.github/workflows/add-remove-new-fulcio.yaml

```diff
@@ -24,13 +24,15 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.25.x
+          - v1.25.x
 
         leg:
-        - fulcio-key-rotation
+          - fulcio-key-rotation
 
         go-version:
-        - 1.18
+          - 1.18
+          - 1.19
+          - '1.20'
```

Comment:
@vaikas what is our golang version support policy

not sure if it is worthwhile to test all the go versions

Reply:
if verify is only running with 1.20, i say we should only support the latest? and drop 1.18 and 1.19

Reply:
at least the latest then, i guess

Reply:
Yeah, good point @k4leung4
Since we do release artifacts and test with those, probably not that important to be testing with various versions of go.